### PR TITLE
Bunch of new stuff for the studies done over the Summer

### PR DESCRIPTION
**Pull request by gartrog**
Also includes a new README

### CODE REVIEW

**Comment by gartrog** [comment]
Wait a bit before merging: once all my other PR are merged I will update the README to point back to the correct git repos (and not to my forks)

File: FCCSW_ecal/runTopoAndSlidingWindowAndCaloSim.py [hunk 1]
```diff
@@ -192,7 +192,7 @@
 # generate noise for each cell
 if addNoise:
     #ecalBarrelNoisePath = "/afs/cern.ch/user/b/brfranco/work/public/Fellow/FCCSW/FCCSW_201207_geometry/LAr_scripts/geometry/noise_capa/elecNoise_ecalBarrelFCCee.root"
-    ecalBarrelNoisePath = "/afs/cern.ch/user/b/brfranco/work/public/Fellow/FCCSW/210927/LAr_scripts/geometry/noise_capa_220216/elecNoise_ecalBarrelFCCee.root"
+    ecalBarrelNoisePath = os.environ['FCCBASEDIR']+"/LAr_scripts/geometry/noise_capa_220216/elecNoise_ecalBarrelFCCee.root"
```
[the remark this answers is not in the document]
Comment (gartrog) [reply FCCSW_ecal/runTopoAndSlidingWindowAndCaloSim.py:195]: Yes, putting it in the git repo would be nice. The FCC eos repository is not always easily accessible from non-CERN locations. Where would you put it in the repo ?

[the remark this answers is not in the document]
Comment (BrieucF) [reply FCCSW_ecal/runTopoAndSlidingWindowAndCaloSim.py:195]: Can you create a folder named `data` at the root of this repository, put it there and use a relative path pointing to it?

[the remark this answers is not in the document]
Comment (gartrog) [reply FCCSW_ecal/runTopoAndSlidingWindowAndCaloSim.py]: Done

**Pull request merged**